crates/sargon/src/types/samples/access_controller_address_samples.rs

```diff
@@ -1,5 +1,6 @@
 use crate::{
```

Comment:
this should probably have been `crate::prelude` actually. But in SOME cases we must use paths, because we have marked some symbol as `pub(super)` which means that all file that module can access it. So it is not publically re-exported at `crate::prelude` level.